pkg/oracles/README.md

```diff
@@ -0,0 +1,91 @@
+# <img src="../../logo.svg" alt="Balancer" height="128px">
+
+# Balancer V3 Oracles
+
+This package contains oracle implementations for Balancer V3 pool types that determine the Balancer Pool Token (BPT) price of each pool. These oracles provide price feeds that are compatible with Chainlink's AggregatorV3Interface, making them suitable for use in DeFi protocols that require reliable price data.
+
+## Overview
+
+The oracles package provides three main oracle implementations:
+
+- **E-CLP Oracle** (`EclpLPOracle.sol`) - For Elliptical Curve Liquidity Provider pools
+- **Stable Oracle** (`StableLPOracle.sol`) - For Stable pools (like Curve-style pools)
+- **Weighted Oracle** (`WeightedLPOracle.sol`) - For Weighted pools (like Uniswap V2-style pools)
```

Comment:
```suggestion
- **Stable Oracle** (`StableLPOracle.sol`) - For V3 Stable pools
- **Weighted Oracle** (`WeightedLPOracle.sol`) - For V3 Weighted pools
```

I'd just remove these references to avoid confusion, since these really just work within Balancer V3. Maybe also add a link to the pool packages?

Reply:
Oh man! I just tested creating this with AI, read the doc quickly and it made sense, but I let it pass, sorry. Totally, I'll remove it and add the links